### ISSUE TITLE
toco convert to lite model incorrectly with slim.batch_norm not follows after conv.

### ISSUE THREAD

**Issue by apollo-time** [tensorflow/tensorflow#17321]
### System information
- **OS Platform and Distribution Linux Ubuntu 14.04**:
- **TensorFlow installed from binary**:
- **TensorFlow version 1.6.rc1**:
- **Python version 3.6**: 
- **Have I written custom code N/A**:
- **Bazel version N/A**:
- **CUDA/cuDNN version N/A**:
- **GPU model and memory N/A**:
- **Exact command to reproduce N/A**:

I create simple model with slim and convert lite model using toco
```python
      input = tf.placeholder(tf.float32, (None, 120, 120, 3), 'input')
      net = input
      net = slim.conv2d(net, 12, 7, stride=2, scope='conv1')
      net = slim.batch_norm(net)
      net = slim.max_pool2d(net, 3, stride=2, padding='SAME')
      net = slim.batch_norm(net)
      ...
```
The converted lite model from it can't allocate tensor with error 
`tensorflow\contrib\lite\kernels\mul.cc:48 NumDimensions(input1) != NumDimensions(input2) (4 != 1)`

but if I insert conv2d layer befor batch_norm, the result is ok.
```python
      input = tf.placeholder(tf.float32, (None, 120, 120, 3), 'input')
      net = input
      net = slim.conv2d(net, 12, 7, stride=2, scope='conv1')
      net = slim.batch_norm(net)
      net = slim.max_pool2d(net, 3, stride=2, padding='SAME')
      net = slim.conv2d(net, 12, 7, stride=2, scope='conv111')
      net = slim.batch_norm(net)
      ...
```
I had try insert relu and the other layers before batch_norm, but it can't allocate tensor when previous layer is not conv.
How can I create batch_norm after any layer?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by coutner** [comment]
do you solve the problem now?

**Comment by apollo-time** [comment]
@coutner not yet!

**Comment by tensorflowbutler** [comment]
Nagging Assignee @petewarden: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @petewarden: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @petewarden: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @petewarden: It has been 75 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @petewarden: It has been 90 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @petewarden: It has been 105 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tofulawrence** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
